pkgs/by-name/aq/aqualung/package.nix

```
@@ -0,0 +1,187 @@
+{
+  lib,
+  stdenv,
+  autoreconfHook,
+  pkg-config,
+  fetchFromGitHub,
+  nix-update-script,
+
+  # Mandatory
+  gtk3,
+  libxml2,
+
+  # Optional
+  enableLadspa ? true,
+  lrdf,
+  enableCdda ? true,
+  libcdio,
+  libcdio-paranoia,
+  enableCddb ? true,
+  libcddb,
+  enableSrc ? true,
+  libsamplerate,
+  enableIfp ? false, # FIXME: find the right dependency if it exists in nixpkgs
+  enableLua ? true,
+  lua,
+
+  # File I/O
+  enableSndfile ? true,
+  libsndfile,
+  enableFlac ? true,
+  flac,
+  enableVorbis ? true,
+  libvorbis,
+  enableSpeex ? true,
+  liboggz,
+  speex,
+  enableMpeg ? true,
+  libmad,
+  enableLame ? true,
+  lame,
+  enableMod ? true,
+  libmodplug,
+  enableMpc ? true,
+  libmpcdec,
+  enableMac ? true,
+  monkeysAudio,
+  enableWavpack ? true,
+  wavpack,
+  enableLavc ? true,
+  ffmpeg,
+
+  # Output
+  enableSndio ? true,
+  sndio,
+  enableAlsa ? true,
+  alsa-lib,
+  enableJack ? true,
+  libjack2,
+  enablePulse ? true,
+  pulseaudio,
+}:
+stdenv.mkDerivation rec {
```

Use `finalAttrs` to make the package aware of `overrideAttrs`
```suggestion
stdenv.mkDerivation (finalAttrs: {
```